common/content/options.js

```diff
@@ -41,7 +41,7 @@ const Option = Class("Option", {
 
         // add no{option} variant of boolean {option} to this.names
         if (this.type == "boolean")
-            this.names = array([name, "no" + name] for (name in values(names))).flatten().__proto__;
+            this.names = util.Array.flatten(Array([name, "no" + name] for (name in values(names)))).__proto__;
```

Comment:
I assumed it was necessary because of the original `array(...)`, which I took for a misspelled `Array(...)`. Maybe it's not necessary and we could get a `__proto__` from the call to `Array.flatten(...)`. I did not try it without the inner Array, as I am not familiar with the codebase and did not know how `util.Array.flatten` operates.